config/install/field.storage.node.field_starships.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_starships
+field_name: field_starships
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node_type
```

Comment:
This needs to reference "Content" instead of "Content Type" in the field settings.
<img width="353" alt="people_starships_field_storage" src="https://github.com/notS0Solo/swapimigration/assets/1434979/485881d9-2b93-4698-a2ad-0a305d722947">
Once that change is made, update the field configuration to reference the "Starships" content type as well.

Reply:
Done and ready for review.